README.md

```diff
@@ -64,6 +67,12 @@ success_comment: "🎉 The action successfully merged all branches with the dev
 # Comment string that will be shown in the Pull request on failure.
 # Default: '' (using a default message)
 failure_comment: "⚠️ Something went wrong."
+# Label string that will be shown on the Pull request on success.
+# Default: successful
+success_label: 'successful'
```

Comment:
```suggestion
success_label: 'successful dev deployment'
```

better default?

Reply:
The idea is good, but I would leave it at the label, because you don't necessarily have to merge to the dev branch and then that can be irritating.